next-cmake/CMakeLists.txt

```diff
@@ -157,6 +172,20 @@ if(HIPBLASLT_ENABLE_HOST)
             ${CMAKE_DL_LIBS}
     )
 
+    if(HIPBLASLT_ENABLE_ASAN)
```

Comment:
Might want to eventually have a helper like `hipblaslt_target_configure_sanitizers(target)` or something.